docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -1,42 +1,116 @@
-////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
+//Only enable one of the following attributes at the same time
+// enable this attribute to show downstream markup and content, must be disabled for upstream doc builds
+//:downstream:
```

Comment:
@inoxx03 Do you mind me asking the process for uncommenting this attribute? How does it work downstream to uncomment `:downstream:` and comment `:upstream:`? 

It seems like that will require manual intervention for the adoc file or a bit of scripting. If you plan to manually edit the adoc file to enable/disable attributes anyway then perhaps some of the things you've added here can be left out of community.

Reply:
@oraNod That's true! While this makes it easier to test and demo the content filtering, it complicates the downstreaming process a lot more that I initially realized. I would like to avoid manual intervention if possible, neither do I want to rely on scripting too much for downstream-specific additions, because it creates maintenance overhead. I'll revert the conditionals to using one attribute (`downstream`). The community content is going to be the default and downstream-only sections are going to be conditionalized. That way we can include the `:downstream:` attribute in the product-specific `attributes.adoc` file and avoid the need to modify the guide before downstreaming. WDYT?

Reply:
Sounds good.